src/ScheduledJobs/Contracts/IParticipantRegistrationDynamoDbRepository.cs

```diff
@@ -10,7 +9,5 @@ public interface IParticipantRegistrationDynamoDbRepository
     {
         IAsyncEnumerable<Participant> GetAllAsync(
             [EnumeratorCancellation] CancellationToken cancellationToken = default);
```

Comment:
Remove `[EnumeratorCancellation]` from the interfaces, keep/add it on the implementation methods.